packages/amplify-frontend-javascript/lib/frontend-config-creator.js

```diff
@@ -603,4 +607,20 @@ function getPlaceIndexConfig(placeIndexResources) {
   return placeIndexConfig;
 }
 
-module.exports = { createAWSExports, getAWSExports, createAmplifyConfig, deleteAmplifyConfig, generateAwsExportsAtPath };
+function getGeofenceCollectionConfig(geofenceCollectionResources) {
+  let defaultGeofenceCollection = '';
+  const geofenceCollectionConfig = {
+    items: [],
+  };
+  geofenceCollectionResources.forEach(geofenceCollectionResource => {
+    const geofenceCollectionName = geofenceCollectionResource.output.Name;
+    geofenceCollectionConfig.items.push(geofenceCollectionName);
+    if (geofenceCollectionResource.isDefault) {
```

Comment:
is this guaranteed to be true for some collection?

Reply:
Yes. In the walkthrough, we make sure that removing a default resource or updating it to be non-default will require the developer to choose another resource as default. There should be checks on the library side to throw appropriate error in case the default does not exist.